test_utils/src/lib.rs

```diff
@@ -550,12 +551,47 @@ pub fn make_optimal_certificates(
     make_certificates(start, stop, initial_parents, keys, 0.0)
 }
 
-pub fn make_certificates(
+// Outputs rounds worth of certificates with optimal parents, signed
```

Comment:
I would suggest to specify on the comment whether the `stop` is inclusive or not

Reply:
Excellent idea, but instead of doing that I'll just convert those ops to use a [`Range`](https://doc.rust-lang.org/std/ops/struct.Range.html)